websites/Y/YouTubetoMP3/dist/metadata.json

```diff
@@ -0,0 +1,19 @@
+ {
+    "$schema": "https://schemas.premid.app/metadata/1.3",
+    "author": {
+      "name": "Lena the fauli",
+      "id": "680778360871190600"
+    },
+    "service": "YouTubetoMP3",
+    "altnames": ["YouTube Converter"],
+    "description": {
+      "en": "Downloading Youtube Videos"
+    },
+    "url": "youtubetomp3music.com",
+    "version": "1.0.0",
+    "logo": "https://cdn.discordapp.com/attachments/805925040330637332/867383549194928128/Download_1.png",
+    "thumbnail": "https://media.discordapp.net/attachments/805925040330637332/867383901726048287/featured-image-youtube-downloader.png?width=757&height=473",

```

Comment:
upload to imgur

Reply:
Sorry uhm can you edit it? i have no idea how xd 


"logo": "https://imgur.com/puEzTVZ",
"thumbnail": "https://imgur.com/0B8MiPN",

Reply:
Edit the file in your editor, commit, push, and it should be updated.

Reply:
thx c: